src/mainboard/seeed/beaglev/DIT

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -x
+set -e
+
+echo empty sudo so we do not need another password prompt
+sudo date
+
+make
+
+echo "power on, hold down reset, once jh7100 starts"
+
+dd if=target/riscv64imac-unknown-none-elf/release/image.bin  of=image.bin bs=80K count=1
+
+ls -l image.bin
+sudo strace -o shit -f -s 1024 -v \
```

Comment:
Why are you running it with strace?

Reply:
it's a terrible program. It hangs, and ... well, lots of things. I'll probably be removing this file at some point.